types/google-cloud__storage/index.d.ts

```diff
@@ -5,7 +5,340 @@
 
 /// <reference types="node" />
 
-export * from "./bucket";
-export * from "./channel";
-export * from "./file";
-export * from "./storage";
+import { ReadStream, WriteStream } from "fs";
+
+type PromiseLibrary<T> = () => PromiseLike<T>;
+
+declare namespace Storage {
+    /**
+     * A bucket in the cloud.
+     */
+    class Bucket {
+        constructor(storage: Storage, name: string);
+        acl: Storage.Acl;
+        combine(sources: string[] | File[], destination: string[] | File[]): Promise<[File, Storage.ApiResponse]>;
+        create(config?: BucketConfig): Promise<[Bucket, Storage.ApiResponse]>;
+        createChannel(id: string, config: ChannelConfig): Promise<[Channel, Storage.ApiResponse]>;
+        delete(): Promise<[Storage.ApiResponse]>;
+        deleteFiles(query?: BucketQuery): Promise<void>;
+        exists(): Promise<[boolean]>;
+        file(name: string, options?: BucketFileOptions): File;
+        get(options?: BucketGetOptions): Promise<[Bucket, Storage.ApiResponse]>;
+        getFiles(query?: BucketQuery): Promise<[File[]]>;
+        getFilesStream(query?: BucketQuery): ReadStream;
+        getMetadata(): Promise<[BucketMetadata, Storage.ApiResponse]>;
+        id: string;
+        makePrivate(options?: BucketPrivacyOptions): Promise<[File[]]>;
+        makePublic(options?: BucketPrivacyOptions): Promise<[File[]]>;
+        metadata: BucketMetadata;
+        name: string;
+        setMetadata(metadata?: BucketMetadata): Promise<[Storage.ApiResponse]>;
```

Comment:
[`setLabels`](https://googlecloudplatform.github.io/google-cloud-node/#/docs/storage/1.2.0/storage/bucket?method=setLabels) seems to be missing.